src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -64,6 +64,25 @@ public static MonikerList GetZoneLevelMonikers(this MarkdownObject obj)
             return default;
         }
 
+        public static string? GetTabbedConceptual(this MarkdownObject obj)
+        {
+            foreach (var parent in obj.GetPathToRootInclusive())
+            {
+                if (parent is TabContentBlock content)
+                {
+                    var group = (TabGroupBlock)content.Parent;
+                    foreach (var item in group.Items)
+                    {
+                        if (item.Content == content)
+                        {
+                            return string.IsNullOrEmpty(item.Condition) ? item.Id : item.Id + "/" + item.Condition;
+                        }
+                    }
+                }
+            }
```

Comment:
Is it possible to add a backward reference from `TabContentBlock` to `TabItemBlock`, so we don't have to iterate through them?

Reply:
Great idea, worth researching the possibility.